source/technical/shares.rst

```diff
@@ -0,0 +1,234 @@
+With the service, you can create a shared file system, mount to the bare metal instances, and manage some of its properties, such as visibility.
+
+  .. hint::
+  
+    Chameleon shared file system service is currently backed by a CephFS. Same as our :ref:`object store <object-store>` service, the data is
+    replicated and the replication should provide good availability in case of hardware failures.
+
+    *Difference between shared file system and object store*
+	You can choose either shared file system or object store to store, manage, and share your data with your collaborators. The object store
+	is suitable for storing large objects at scale, but isn’t suitable for transactional data, as objects are immutable and updated in their
+	entirety. On the other hand, Chameleon shared file system maps the CephFS using file-based NFS protocol to the bare metal instances, and
+	the NFS protocol manages locking and data integrity processes required to provide multiple concurrent access to data. 
+
+The shared file system service is available at `CHI@UC <https://chi.uc.chameleoncloud.org/>`_ and `CHI@TACC <https://chi.tacc.chameleoncloud.org/>`_.
+Each region has its own service and the shares created at one region are not available to the other. As all other Chameleon services, you can create
+and manage your shares using both GUI and CLI.
+
+.. _storage_vlans:
+
+Storage VLANs
+================================
+
+To provide isolation among shares created by different projects, accessing a share requires the storage VLANs, which are special VLANs you can
+reserve to use. When reserving the storage VLANs, add `usage_type=storage` to the resource properties. To learn more about reserving VLANs, read
+the :ref:`reservations documentation <reservations>`. All bare metal instances that are created on the storage VLAN have access to all the project
+shares.
+
+Shares
+================================
+
+Visibility
+--------------------------------
+
+Shares are owned by the project. By default, all shares have `private` visibility and can only be listed and accessed within your project.
+All bare metal instances owned by the project have the read and write permissions to the project’s shares. You can also make your shares `public`.
+All Chameleon users and projects can list public shares, and with a storage VLAN, all projects have read-only access to a public share.
+
+Accessibility
+--------------------------------
+
+A share is a pre-allocated storage space at a CephFS. You can :ref:`mount your shares to your bare metal instances via NFS protocol <mount-share>`.
+The accessibility of the shares are controlled internally by the reservation service. You are not allowed to edit the access rules of a share.
+
+Quotas
+--------------------------------
+
+We do not charge SUs for the storage spaces of your shares. However, we do limit the total size and the number of shares you can create within
```

Comment:
"the storage spaces" - "the" is actually appropriate here, at least to me. So... it's a hard thing to generalize I guess.